### ISSUE TITLE
Filter names

### ISSUE THREAD

**Issue by kidvicious** [pauljeannot/SnapSliderFilters#18]
Is there a way to display the filter names on swiping?

**Comment by pauljeannot** [comment]
Hey @kidvicious ! What do you mean by that ?
Like a UILabel on the screen that slides with the filter (as a Sticker would) ?

**Comment by kidvicious** [comment]
yes like in Instagram stories..name for a while and disappears..
